highlight/highlight
# Provide AI suggestions for resolving errors

## Summary

Adds a feature where users can get a suggestion from the OpenAI API for resolving an error.

https://www.loom.com/share/3dde3fb532ba4b03932c0f733c6c133e?sid=7e86bcdf-293c-4eee-9319-75ca25c2ee53

## How did you test this change?
1. Load an error as a free tier workspace
- [ ] The prompt for Harold is displayed
2. Click on the button to initiate a response
- [ ] The response is returned and displayed
3. Click to generate another response
- [ ] A new response is returned and displayed

## Are there any deployment considerations?

Should monitor usage of the feature, any errors, and costs for OpenAI.

## TODOs

- [x] Finalize design w/ @julian-highlight: [Figma designs](https://www.figma.com/file/rdBPNzn7Klk6RG1LHUCE5B/Screen-Designs?type=design&node-id=9300-227758&mode=design&t=sr7AEVFfpCmBDxmM-0)
- [X] Finalize copy w/ @jay-khatri
- [x] Add secret to Doppler
- [x] Set spending limit on OpenAI 

### Design Requirements

* Collecting user feedback
* CTA to upgrade
* Usage limit
